openshift/release/resolve.sh

```diff
@@ -21,7 +21,7 @@ function resolve_file() {
   # 1. Rewrite image references
   # 2. Update config map entry
   # 3. Replace serving.knative.dev/release label.
-  sed -e "s+serving.knative.dev/release: devel+serving.knative.dev/release: \"v1.1.2\"+" \
-      -e "s+app.kubernetes.io/version: devel+app.kubernetes.io/version: \"v1.1.2\"+" \
+  sed -e "s+serving.knative.dev/release: devel+serving.knative.dev/release: \"v1.2.0\"+" \
```

Comment:
Could you please remove this sed `"s+serving.knative.dev/release: devel+serving.knative.dev/release: \"v1.2.0\"+"`?
The label was removed from the upstream so this sed does not do anything.

Reply:
Ok will do

Reply:
If I remove that part, there will be no checking against the version in manifests do we want that before the bug at the operator side is fixed?

Reply:
The operator's version check for the version label does not make sense at all. They added the version check for their `spec.version` feature but we don't use it so just it is alright to ignore it.

Reply:
ok I will remove it :shrug:

Reply:
Hmm...  I think you misunderstand something. Are you thinking that https://github.com/openshift/knative-serving/blob/release-next/openshift/release/knative-serving-ci.yaml is used by S-O?
The knative-serving-ci.yaml is just a temporary manifest only for the CI in this repository.

If you want to add the label you need to add the [upstream's manifest](https://github.com/knative/serving/blob/6ec4509c73094850c3d517e1c8448aee0dab91cc/config/core/deployments/controller.yaml)  or patch it in S-O repo `openshift-knative-operator/hack/update-manifests.sh`.
With said, I still don't understand why you want to add the `serving.knative.dev/release` label which upstream replaced it with `app.kubernetes.io/version` label .

Reply:
Ok that is why I asked for the pointer missed that PR. I will remove it then and we shall keep it elsewhere until 1.4.